api/tests/routes/backoffice/multiple_offers_test.py

```diff
@@ -27,6 +28,8 @@
     pytest.mark.backoffice,
 ]
 
+# TODO : adapt test to real cases (offers with ean without products, and only one product per ean)
```

Comment:
Mets ton nom et et la date du jour quand tu rajoutes un TODO ;)
Et explique aussi à partir de quand on peut le corriger.
Du genre `# TODO : remove products from manual offers when products will be restricted to synchronized offers only`

Reply:
Ajouté